plone/app/vocabularies/catalog.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+import json
```

Comment:
This import should go after `import itertools`.
Not a big deal, but imports are already sorted and would be nice to keep them sorted